var/spack/repos/builtin/packages/r-rmutil/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RRmutil(RPackage):
+    """rmutil: Utilities for Nonlinear Regression and Repeated
+       MeasurementsModels"""
+
+    homepage = "http://www.commanster.eu/rcode.html"
+    url      = "https://cran.r-project.org/src/contrib/rmutil_1.1.3.tar.gz"
+    list_url = "https://cran.r-project.org/src/contrib/Archive/rmutil"
```

Comment:
See #12019. We are standardizing on cloud.r-project.org

Reply:
Updated; thanks for the heads up!